interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -116,10 +116,10 @@ namespace cling {
     typedef llvm::SmallVector<DelayCallInfo, 64> DeclQueue;
     typedef llvm::SmallVector<Transaction*, 2> NestedTransactions;
 
-    ///\brief The list of cxxmodules, which is collected by DeserializationListener
+    ///\brief The list of clang decls, which is collected by DeserializationListener
     /// and will be used to load corresponding libraries.
     ///
-    std::vector<clang::Module*> m_CxxModules;
+    std::vector<const clang::Decl*> m_ClangDecls;
```

Comment:
How's that different from the `DeclQueue`? These are all deserialized `Decl`s, right? Then please consider updating the member name to reflect that, e.g. `m_DeserializedDecls`.